pkg/util/mount/mount.go

```diff
@@ -18,14 +18,11 @@ limitations under the License.
 // an alternate platform, we will need to abstract further.
 package mount
 
-// Each supported platform must define the following flags:
-//   - FlagBind: specifies a bind mount
-//   - FlagReadOnly: the mount will be read-only
 type Interface interface {
-	// Mount wraps syscall.Mount().
-	Mount(source string, target string, fstype string, flags uintptr, data string) error
-	// Umount wraps syscall.Mount().
-	Unmount(target string, flags int) error
+	// Mount mounts source to target as fstype with given options.
+	Mount(source string, target string, fstype string, options []string) error
+	// Unmount unmounts target with given options.
+	Unmount(target string, options []string) error
```

Comment:
Theoretically force umount exists.  In practice, I have never seen it do what the caller hoped it would do.  But fine.